EveripediaNetwork/ep-ui
# Preload state for images

# Add preload states for images

_Images don't have a preload state when selecting category tabs, they appear empty when other content loads, preload skeleton state has been added to ensure a good user experience when viewing each category tab_

## How should this be tested?

1. On the homepage, scroll to the categories section
2. Click on each tab to see the preload state

closes https://github.com/EveripediaNetwork/issues/issues/3057
